packages/webapp/components/layouts/AccountLayout/common.tsx

```diff
@@ -95,7 +95,7 @@ export const accountPage: Record<AccountPage, AccountPageProps> = {
 export const AccountPageContent = classed(
   'div',
   pageBorders,
-  'flex flex-col w-full max-w-[40rem] tablet:border-l mr-auto',
+  'flex flex-col w-full max-w-[calc(100vw-19.75rem)] laptop:max-w-[calc(100vw-36rem)] laptopL:max-w-[40rem] tablet:border-l mr-auto',
```

Comment:
Should this really grow as wide?
Perhaps double check with design on that.

Reply:
I agree it should actually be 640px, but they insisted back then to have this new layout size. With that, I messaged in the channel to confirm with them.

Reply:
The code `laptopL:max-w-[40rem]` should handle the larger screens setting a specific width.